### PR TITLE
Switch Headline font to Geomanist

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -3,6 +3,10 @@
   "env": {
     "USE_AUTH": "true",
     "USE_HELMET": "true",
+    "SYNC_STORYBLOK_ON_RELEASE": "false",
+    "REDIS_URL": {
+      "required": true
```
Comment (palmenhq) [inline app.json]: Maybe we should spin up our own redis for each pr instance instead? So we don't mix the caches, i smell weird issues from using the same cache for multiple servers 😬

Comment (hedvigoscar) [reply app.json]: +1

Comment (hedvigoscar) [reply app.json]: Fixed 😬

**Comment by hedvigoscar** [comment]
@palmenhq Whenever you have time 😬

File: app.json [hunk 2]
```diff
@@ -16,7 +17,12 @@
       "required": true
     }
   },
-  "addons": [],
+  "addons": [
+    {
+      "plan": "heroku-redis",
+      "as": "REDIS"
```
Comment (palmenhq) [inline app.json]: Should we perhaps specify the plan? Preferably hobby dev or something

**Comment by hedvigoscar** [comment]
@palmenhq Done :+1:

**Pull request merged**